File: 2419. Longest Subarray With Maximum Bitwise AND.java
class Solution {
    public int longestSubarray(int[] nums) {
        int max = nums[0];
        for (int i : nums) max = Math.max(i, max);
        int count = 0;
        int answer = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != max) count = 0;
            else count++;
            answer = Math.max(answer, count);
        }
        return answer;
    }
}

File: 2419. Longest Subarray With Maximum Bitwise AND.go
func longestSubarray(nums []int) int {
	m := nums[0]
	for _, v := range nums {
        m = max(m, v)
    }
	count, answer := 0, 1
	for _, v := range nums {
		if v == m {
			count++
		} else {
			count = 0
		}
		answer = max(answer, count)
	}
	return answer
}